cli-core/reportGlobal.js

```diff
@@ -132,6 +138,21 @@ async function create_global_report(reports,options){
     };
     
     if (progressBar) progressBar.tick()
```

Comment:
I think it wouldn't be too problematic for the progress bar to reach 100% before preparing data for the report (analysis is already finished at this point), and it would allow to log the error directly in the `catch` block instead of having to create a dedicated variable and `if` block afterwards. The code would be a lot cleaner and easier to read.